<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="149" started-at="2021-04-24T17:28:56Z" finished-at="2021-04-24T17:28:56Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="149" started-at="2021-04-24T17:28:56Z" finished-at="2021-04-24T17:28:56Z">
      <class name="dataDrivenTesting.DDT_AddNewEmployeeBYReadingTheDataFromXLfile">
        <test-method status="SKIP" signature="postNewEmployees(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:dataDrivenTesting.DDT_AddNewEmployeeBYReadingTheDataFromXLfile@29ee9faa]" name="postNewEmployees" duration-ms="0" started-at="2021-04-24T22:58:56Z" data-provider="empDataProvider" finished-at="2021-04-24T22:58:56Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NoSuchFieldError: DEFAULT]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NoSuchFieldError: DEFAULT
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:49)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:115)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:612)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1316)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1044)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1134)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NoSuchFieldError: DEFAULT
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:162)
	at UtilsPackage.XLUtils.getRowCount(XLUtils.java:31)
	at dataDrivenTesting.DDT_AddNewEmployeeBYReadingTheDataFromXLfile.getEmpData(DDT_AddNewEmployeeBYReadingTheDataFromXLfile.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:55)
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
	... 22 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postNewEmployees -->
      </class> <!-- dataDrivenTesting.DDT_AddNewEmployeeBYReadingTheDataFromXLfile -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
